gnucash/import-export/aqb/dialog-ab-trans.c

```diff
@@ -153,6 +153,9 @@ gboolean gnc_ab_trans_isSEPA(GncABTransType t)
     switch (t)
     {
     case SEPA_TRANSFER:
+#if (AQBANKING_VERSION_INT >= 60400)
+    case SEPA_INTERNAL_TRANSFER:
```

Comment:
Should we mention it in https://github.com/Gnucash/gnucash/blob/maint/README.dependencies#L96 and add a note to https://wiki.gnucash.org/wiki/Dependencies#Libraries.2FDeps?

Reply:
No.